includes/reports/data/customers/class-top-five-customers-list-table.php

```diff
@@ -36,9 +36,9 @@ class Top_Five_Customers_List_Table extends \EDD_Customer_Reports_Table {
 	 */
 	public function get_data() {
 		$data  = array();
-		$taxes = Reports\get_filter_value( 'taxes' );
+		$taxes = Reports\get_taxes_excluded_filter();
 
-		if ( false === $taxes['exclude_taxes'] ) {
+		if ( ! $taxes ) {
```

Comment:
Lets keep the strict type check with:

```suggestion
		if ( false === $taxes ) {
```